meerk40t/core/planner.py

```diff
@@ -882,7 +882,7 @@ def origin():
 
 def unlock():
     yield "rapid_mode"
-    yield "unlock"
+    yield "unlock_rail"
```

Comment:
This command was changed to unlock_rail since .lock was sometimes a thread.Lock() in some devices.